linters/perltidy/perltidy.test.ts

```diff
@@ -0,0 +1,3 @@
+import { linterFmtTest } from "tests";
+
+linterFmtTest({ linterName: "perltidy", namedTestPrefixes: ["basic"] });
```

Comment:
```suggestion
import { linterFmtTest } from "tests";
import { skipOS }  from "tests/utils";

linterFmtTest({ linterName: "perltidy", namedTestPrefixes: ["basic"], skipTestIf: skipOS(["darwin"]) });
```

Reply:
Not skipped if macOS is configured in GHA workflow.